pkgs/by-name/or/ory/package.nix

```
@@ -3,32 +3,38 @@
   buildGoModule,
   fetchFromGitHub,
   installShellFiles,
+  nix-update-script,
+  versionCheckHook,
 }:
 
-buildGoModule rec {
+buildGoModule (finalAttrs: {
   pname = "ory";
-  version = "0.3.4";
+  version = "v1.1.0";
 
   src = fetchFromGitHub {
     owner = "ory";
     repo = "cli";
-    tag = "v${version}";
-    hash = "sha256-q7+Fpttgx62GbKxCCiEDlX//e/pNO24e7KhhBeGRDH0=";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-+Su2FIuCb2vpPW/OCOTzqQOZPpY9gGRbwylSepLh2hk=";
   };
 
   nativeBuildInputs = [
     installShellFiles
   ];
 
-  subPackages = [ "." ];
-
   env.CGO_ENABLED = 1;
 
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/ory/cli/buildinfo.Version=${finalAttrs.version}"
```

```suggestion
    "-X github.com/ory/cli/buildinfo.Version=v${finalAttrs.version}"
```

does this work? Anywhere else that could be patched, or is there some environment variable we could set?